cli/tests/test_parallelize_command.rs

```diff
@@ -241,9 +242,19 @@ fn test_parallelize_failure_disconnected_target_commits() {
     ◉  000000000000
     "###);
 
-    insta::assert_snapshot!(test_env.jj_cmd_failure(
-        &workspace_path, &["parallelize", "description(1)", "description(3)"]),@r###"
-    Error: Cannot parallelize since the target revisions are not connected.
+    let (stdout, stderr) = test_env.jj_cmd_ok(
```

Comment:
nit: the test name should change since this is no longer a failure case per-se.

Reply:
will do. this is just a draft to show how it will work. i also plan to add library-level tests of `transform_descendants()`.

Reply:
Done.